web/src/main/java/se/inera/intyg/webcert/web/service/facade/impl/certificatefunctions/GetUserResourceLinksImpl.java

```diff
@@ -29,12 +29,9 @@
 import se.inera.intyg.webcert.web.web.controller.facade.dto.ResourceLinkTypeDTO;
 
 @Service
+@NoArgsConstructor
```

Comment:
Isn't default constructor "no args" and this is redundant?

Reply:
Removed annotation